### ISSUE TITLE
[bot] Label docs PRs with t-shirt size (S, M, L) on PR creation

### ISSUE THREAD

**Issue by ptgott** [gravitational/teleport#14214]
This is a feature request for our GitHub Actions workflow bot. As a Technical Writer who needs to review a lot of docs PRs, I would like to be able to filter these PRs by size in order to plan my review work. For example, if I know that most docs PRs I need to review are `size:S` (<100 lines of deletions plus additions), I can be confident that I can clear my review backlog in, say, under an hour. However, if I have a few PRs that are `size:L` (say, > 500 lines), then I'll know that I need to carve out extra time to get through them. 

This logic shouldn't apply to backport PRs, since we generally approve these as long as the original commit was cherry-picked cleanly.

This change might also be helpful for reviewing code PRs in this repo, but I can't speak to this since I generally only review docs PRs.

After #13350, adding labels based on PR size should be pretty straightforward.

## What problem does this solve?

This will help anyone who reviews docs PRs (and possibly code PRs) organize their time.

## If a workaround exists, please include it.
My workaround has been to check each PR individually and, if it's too large to review within my allotted time frame, check another PR. An alternative could be to write a script that queries the GitHub API for a list of PRs and sorts them by size.

**Comment by zmb3** [comment]
FYI @ptgott this is completed, but it uses the same heuristics for sizing in both docs and code PRs. If you find that things are mislabeled and you want to tweak the sizing let me know.

**Comment by ptgott** [comment]
@zmb3 Thanks a lot for this! 🎉
